builders/errors.go

```diff
@@ -0,0 +1,18 @@
+package elrond
```

Comment:
why the package is named `elrond`?

Reply:
it was copied from the eth bridge. pushed before finishing the review